nav_app/views.py

```diff
@@ -7,12 +7,17 @@
 @csrf_exempt
 def book_taxi(request):
     if request.method == 'POST':
-        json_data = request.body.decode('utf-8')
-        data = json.loads(json_data)
-        my_model = GetUserInfo(location=data['location'], departure_time=data['departure_time'], email=data['email'])
-        my_model.save()
-        response_data = {'success': True}
-        return JsonResponse(response_data)
+        try:
+            json_data = request.body.decode('utf-8')
+            data = json.loads(json_data)
+            my_model = GetUserInfo(location=data['location'], departure_time=data['departure_time'],
+                                   email=data['email'])
+            my_model.save()
+            response_data = {'success': True}
+            return JsonResponse(response_data)
+        except ValueError as e:
```

Comment:
These changes should not have been part of commit with message "Use .env file". Should have been a separate commit.